custom/src/main/java/co/elastic/otel/dynamicconfig/CentralConfig.java

```diff
@@ -40,30 +40,27 @@ public class CentralConfig {
   }
 
   public static void init(SdkTracerProviderBuilder providerBuilder, ConfigProperties properties) {
-    // TODO flip default when EDOT collector supports op amp
-    boolean startOpAmp = properties.getBoolean("elastic.otel.opamp.start", false);
-    if (!startOpAmp) {
+    String endpoint = properties.getString("elastic.otel.opamp.endpoint");
+    if (endpoint == null || endpoint.isEmpty()) {
+      logger.fine("OpAMP is not enabled");
```

Comment:
```suggestion
      logger.fine("OpAMP is disabled");
```
The negation is likely to not be read and people might read it as "enabled" when it's "not enabled.